### DISCUSSION TITLE
Integrate Pinia into the project

### DISCUSSION THREAD

**Discussion by MaloLebrin** [sidebase/sidebase#44] [Ideas]
What do you think of integrating Pinia the new Powerful view store ?

**Comment by BracketJohn** [comment]
Like it (: If you want to, you can try your hand at it and open a PR for it? 🌟 Else we'll put it on our roadmap.

I've opened and issue for it here: #45

**Reply by MaloLebrin** [reply]
Thank you @BracketJohn, i'll do it when i find some times 😇

**Reply by BracketJohn** [reply]
Thanks ❤️  Looking forward to pushing onwards your PR #51 💯
